app/views/layouts/_header.html.erb

```diff
@@ -4,7 +4,9 @@
     <ul class="nav navbar-nav navbar-right">
       <% if current_user %>
         <ul>
-          <li><%= link_to "Create Business", new_business_path %></li>
+          <% if current_user.business.blank? || current_user.business.new_record? %>
```

Comment:
why using `.new_record` at all? http://apidock.com/rails/ActiveRecord/Base/new_record%3F

why not making sure that every time business is created, it is saved..